warehouse/organizations/models.py

```diff
@@ -158,10 +159,13 @@ class OrganizationTermsOfServiceAgreement(db.Model):
     organization_id: Mapped[UUID] = mapped_column(
         ForeignKey("organizations.id", onupdate="CASCADE", ondelete="CASCADE"),
     )
-    agreed: Mapped[datetime.datetime | None] = mapped_column(TZDateTime)
-    notified: Mapped[datetime.datetime | None] = mapped_column(TZDateTime)
+    revision: Mapped[str]
+    datetime: Mapped[datetime.datetime] = mapped_column(TZDateTime)
```

Comment:
I don't know if it's too late to correct, but naming the column `datetime` made my eyes water. This and the `user_terms_of_service_engagements.datetime` are the only ones in the DB named `datetime`, others use `created` to convey the timestamp without time zone.

It made for visually trying to distinguish what's a Python `datetime` and a SQL column name hard.

Reply:
why would it be too late? df6d77f42c6e8fd01efd6fc506ed572621c11c70